docs/topics/con_springboot-runtime-design-details.adoc

```diff
@@ -1,7 +1,9 @@
-[id='springboot-runtime-details_{context}']
-= {SpringBoot} runtime details
+[id='springboot-runtime-design-details_{context}']
+= {SpringBoot} runtime design details
 
-{SpringBoot} lets you create opinionated Spring-based stand-alone applications. See xref:additional-springboot-resources_{context}[Additional Resources] for a list of documents about {SpringBoot}.
+{SpringBoot} lets you create opinionated Spring-based stand-alone applications.
+See
+link:{link-spring-boot-guide}#additional-springboot-resources_spring-boot[Additional Resources] for a list of documents about {SpringBoot}.
```

Comment:
It's a single sentence, so should be on a single line?

Reply:
Of course, thanks. It's a remnant of an `ifdef` I put there and replaced later.